### ISSUE TITLE
Komga source: AVIF filetype (but would only work on iOS 16+)

### ISSUE THREAD

**Issue by Le-R** [Skittyblock/aidoku-community-sources#321]
Right now Komga server don't have full support to AVIF files, since there's no java library to deal with AVIF files.
But Komga is able to at least store and serve the AVIF files as is, it just need that the client asks the file without the `?convert` option.

Komga dev published an update to the Tachiyomi extension that adds "image/avif" on the supported file types, so when serving an AVIF file, Komga server will not try to convert and just send it as is and tachi will open the pages as normal:
https://github.com/tachiyomiorg/tachiyomi-extensions/pull/16217

On Aidoku, adding the `"image/avif"` on multi.komga/src/lib.rs#L228 would work the same way if the user is on iOS 16, where Apple added native support to AVIF files. But still wouldn't work on iOS 15 and below.

Personally I have been using  for a while a self recompiled version of Komga server, customized to ignores the `?convert` on AVIF files and Aidoku is working fine with those archives.

This change would allow to users using the vanilla Komga server, but the problem is that there's no support to AVIF on iOS15 and below, so I don't know if this could be really acceptable or not.

PS: While the Tachiyomi commit have both AVIF and HEIF, I don't know if HEIF works. The HEIF files I tried to create didn't work even on tachi.

---

**In short, current behavior**:
If open a AVIF file stored on the Komga server, Komga will try to convert from AVIF to png and fail. The image will not open.

**Behaviour after:**
_On iOS 16:_
Komga will send the AVIF file as is and it will work as normal, because Apple added native AVIF support to iOS 16.

_On iOS 15 and below:_
Komga will send the AVIF file as is and it will still not work (unless AVIF support is added to Aidoku itself).
Probably this is the is the main issue/roadblock.

---

Anyway, thanks for the awesome work!

**Comment by Skittyblock** [comment]
komga now a built in source so not necessary
